lib/server.js

```diff
@@ -83,13 +79,17 @@ var PROXY_EVENTS = [
  * @param {Boolean} [options.handleUpgrades=false] - Hook the `upgrade` event
  * from the node HTTP server, pushing `Connection: Upgrade` requests through the
  *  regular request handling chain.
+ * @param {Boolean} [options.onceNext=false] - Prevents calling next multiple
+ *  times
+ * @param {Boolean} [options.strictNext=false] - Throws error when next() is
+ *  called more than once, enabled onceNext option
  * @param {Object} [options.httpsServerOptions] - Any options accepted by
  * [node-https Server](http://nodejs.org/api/https.html#https_https).
  * If provided the following restify server options will be ignored:
  * spdy, ca, certificate, key, passphrase, rejectUnauthorized, requestCert and
  * ciphers; however these can all be specified on httpsServerOptions.
- * @param {Boolean} [options.strictRouting=false] - If set, Restify
- * will treat "/foo" and "/foo/" as different paths.
+ * @param {Boolean} [options.noWriteContinue=false] - prevents
+ *  `res.writeContinue()` in `server.on('checkContinue')` when proxing
```

Comment:
proxying?

Reply:
Hmm, I don't remember where I found this. I just documented it, could you help me with the correct behaviour?